#build/Publish-All.ps1

```diff
@@ -126,7 +135,7 @@ try {
         Write-Output "*** $_"
 
         $siteRoslynArtifactsRoot = Resolve-Path "$roslynArtifactsRoot\$($_.Name)"
-        $branchInfo = ConvertFrom-Json ([IO.File]::ReadAllText("$siteRoslynArtifactsRoot\BranchInfo.json"))
+        $branchInfo = ConvertFrom-Json ([IO.File]::ReadAllText((Resolve-Path "$siteRoslynArtifactsRoot\BranchInfo.json")))

```

Comment:
`$siteRoslynArtifactsRoot` should already be resolved by this point though?

Reply:
Good point! It's related to how PowerShell 6 handles `\`.
Linux/macOS versions of PowerShell 6 doesn't interpret `\` as a path-separator unless used in `Resolve-Path`, so the original code raise error (FileNotFound) in Linux/macOS. Unfortunately, we need to `Resolve-Path`-ize things like we call Path.Combine() in C# for cross-platform.